cfme/infrastructure/provider/__init__.py

```diff
@@ -302,7 +304,7 @@ class Details(CFMENavigateStep):
     prerequisite = NavigateToSibling('All')
 
     def step(self):
-        sel.click(Quadicon(self.obj.name, self.obj.quad_name))
+        self.prerequisite_view.items.get_item(by_name=self.obj.name).click()
```

Comment:
So if I understand this correctly, this could be a quadicon, tile, or list item and it'll still just click it. no need to set the view for navigation. :heart: